nemo/collections/vision/data/megatron/autoaugment.py

```diff
@@ -0,0 +1,270 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Code adapted from https://github.com/DeepVoltaire/AutoAugment.
```

Comment:
MIT License so I think it's okay. But it would still be good to reach out to the authors to ask if they are okay with us adapting their code.